php/php.editor/src/org/netbeans/modules/php/editor/completion/DocRenderer.java

```diff
@@ -391,10 +388,11 @@ private void extractPHPDocBlock() {
             List<PhpElement> inheritedElements = getInheritedElements();
             List<PHPDocBlock> inheritedComments = getInheritedComments(inheritedElements);
 
-            String description = phpDocBlock.getDescription();
+            String description = phpDocBlock == null ? null : phpDocBlock.getDescription();
             description = composeDescription(description, inheritedComments);
 
-            for (PHPDocTag tag : phpDocBlock.getTags()) {
+            List<PHPDocTag> tags = phpDocBlock == null ? new ArrayList<>() : phpDocBlock.getTags();
```

Comment:
Should use `Collections.emptyList()` instead of `new ArrayList<>()`

Reply:
Changed, thank you.